### ISSUE TITLE
Stage 1 Pre-training Loss Objective?

### ISSUE THREAD

**Issue by GloryyrolG** [cure-lab/MotionCraft#6]
Hi @yxbian23 @ailingzengzzz et al.,

thx for ur great contribution! may i ask it seems not clear in the paper, what the loss objective for stage 1 pre-training is?

best,

**Comment by yxbian23** [comment]
> Hi @yxbian23 @ailingzengzzz et al.,
> 
> thx for ur great contribution! may i ask it seems not clear in the paper, what the loss objective for stage 1 pre-training is?
> 
> best,

Thanks for your interest in our work! Currently, our loss is just the MSE loss between the SMPL-X result predicted by the model and the Ground Truth, but we are exploring adding physical loss functions such as foot contact, local positions, local velocities, etc. Stay tuned!
